gui/src/3D/controls/nearMenu.ts

```diff
@@ -150,8 +153,8 @@ export class NearMenu extends VolumeBasedPanel {
 
         this._pinButton.position.copyFromFloats(this._backPlate.scaling.x / 2 + 0.02, this._backPlate.scaling.y / 2, -0.01);
 
-        this._defaultBehavior.followBehavior.minimumDistance = extendSize.length() * 0.5 * this.scaling.length();
-        this._defaultBehavior.followBehavior.maximumDistance = extendSize.length() * 1.5 * this.scaling.length();
+        this._defaultBehavior.followBehavior.minimumDistance = extendSize.length() * 3 * this.scaling.length();

```

Comment:
what is the reason behind the change here? is this something we need to store as a readonly static property?

Reply:
that's a mistake, I'm reverting

Reply:
It is changed again in a more recent PR (#10485), to be a fixed value for the near menu. I will add a parameter in this PR instead